boards/arm64/qemu_cortex_a53/Kconfig.defconfig

```diff
@@ -9,4 +9,11 @@ config BUILD_OUTPUT_BIN
 config BOARD
 	default "qemu_cortex_a53"
 
+if IPM
+
+config IPM_GIC_SGI
+	default y
```

Comment:
Why this is unconditionally enabled? This should be enabled at application level, not board level.